### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by rsy6318** [luost26/score-denoise#10]
In the paper, 20 meshes of PU dataset were used to train, but in the code, there are 40 meshes. Can it influence the result?
I have retrain the network and get worse results.

**Comment by luost26** [comment]
Did you train the model for a long enough time, or did the training converge?

The scale of the point clouds affect the metric values. The same normalization should be apply to get comparable results.

**Comment by ddsediri** [comment]
Hi @luost26, I would like to clarify for the pretrained model (and its results mentioned in the paper), was 20 meshes sampled or was it 40? If only 20 were sampled, is it possible to provide the list of meshes?

Thank you!

**Comment by luost26** [comment]
> Hi @luost26, I would like to clarify for the pretrained model (and its results mentioned in the paper), was 20 meshes sampled or was it 40? If only 20 were sampled, is it possible to provide the list of meshes?
> 
> Thank you!

The training set size is 40. 20 is a mistake in the paper. Sorry about that.

**Comment by ddsediri** [comment]
@luost26, fantastic, that's all good then. Great work and thanks for the clarification!
